try.html

```diff
@@ -694,18 +694,46 @@ <h3 id="miscellaneous"> Miscellaneous </h3>
     <td><img src='/wheelmap/a/2323004600.svg' alt='' /></td>
     <td><code>https://img.shields.io/wheelmap/a/2323004600.svg</code></td>
   </tr>
-  <tr><th data-doc='githubDoc'> GitHub issues: </th>
+  <tr><th data-keywords='GitHub issue' data-doc='githubDoc'> GitHub issues: </th>
     <td><img src='/github/issues/badges/shields.svg' alt=''/></td>
     <td><code>https://img.shields.io/github/issues/badges/shields.svg</code></td>
   </tr>
-  <tr><th data-keywords='GitHub' data-doc='githubDoc'></th>
+  <tr><th data-keywords='GitHub issue' data-doc='githubDoc'></th>
     <td><img src='/github/issues-raw/badges/shields.svg' alt=''/></td>
     <td><code>https://img.shields.io/github/issues-raw/badges/shields.svg</code></td>
   </tr>
-  <tr><th data-keywords='GitHub' data-doc='githubDoc'> label:</th>
+  <tr><th data-keywords='GitHub issue' data-doc='githubDoc'> GitHub closed issues: </th>
+    <td><img src='/github/issues-closed/badges/shield.svg' alt=''/></td>
+    <td><code>https://img.shields.io/github/issues-closed/badges/shields.svg</code></td>
+  </tr>
+  <tr><th data-keywords='GitHub issue' data-doc='githubDoc'></th>
+    <td><img src='/github/issues-closed-raw/badges/shields.svg' alt=''/></td>
+    <td><code>https://img.shields.io/github/issues-closed-raw/badges/shields.svg</code></td>
+  </tr>
+  <tr><th data-keywords='GitHub issue label' data-doc='githubDoc'> label:</th>
     <td><img src='/github/issues-raw/badges/shields/website.svg' alt=''/></td>
     <td><code>https://img.shields.io/github/issues-raw/badges/shields/website.svg</code></td>
   </tr>
+  <tr><th data-keywords='GitHub pullrequest pr' data-doc='githubDoc'> GitHub pull requests: </th>
+    <td><img src='/github/issues-pr/cdnjs/cdnjs.svg' alt=''/></td>
+    <td><code>https://img.shields.io/github/issues-pr/cdnjs/cdnjs.svg</code></td>
+  </tr>
+  <tr><th data-keywords='GitHub pullrequest pr' data-doc='githubDoc'></th>
+    <td><img src='/github/issues-pr-raw/cdnjs/cdnjs.svg' alt=''/></td>
+    <td><code>https://img.shields.io/github/issues-pr-raw/cdnjs/cdnjs.svg</code></td>
+  </tr>
+  <tr><th data-keywords='GitHub pullrequest pr' data-doc='githubDoc'> GitHub closed pull requests: </th>
+    <td><img src='https://img.shields.io/github/issues-pr-closed/cdnjs/cdnjs.svg' alt=''/></td>
```

Comment:
Images in try.html must not point to img.shields.io. Start the URL with `/` please.